build/generate-travis

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
Full disclosure, I might rewrite this in a language I myself know better some day :)

Reply:
no problem, I just didn't want to do it in unmaintainable bash :laughing: